src/uipath/models/attachment.py

```diff
@@ -17,9 +17,14 @@ class Attachment(BaseModel):
         use_enum_values=True,
         arbitrary_types_allowed=True,
         extra="allow",
-        json_encoders={datetime: lambda v: v.isoformat() if v else None},
     )
 
+    @field_serializer("*", when_used="json")
```

Comment:
```suggestioncreation_time
    @field_serializer("creation_time", when_used="json")
```
for models that have just one datetime field maybe we can target that specific value, instead of applying the serializer on all fields